render/hyperscript.js

```diff
@@ -32,13 +32,12 @@ function compileSelector(selector) {
 
 function execSelector(state, vnode) {
 	var attrs = vnode.attrs
-	var children = Vnode.normalizeChildren(vnode.children)
 	var hasClass = hasOwn.call(attrs, "class")
 	var className = hasClass ? attrs.class : attrs.className
 
 	vnode.tag = state.tag
 	vnode.attrs = null
-	vnode.children = undefined
+	vnode.children = Vnode.normalizeChildren(vnode.children)
```

Comment:
Just remove this line. It's not necessary - the normalization is already done prior `execSelector` even being called.